source/docs/03-metadata.xml

```diff
@@ -32,8 +32,7 @@
             <title level="a">Metadata in MEI</title>
             <respStmt>
                <resp>Edited by</resp>
-               <name role="edt">Margrethe Bue</name>
-               <name role="edt">Kristina Richts-Matthaei</name>
+               <name role="edt">Dennis Ried</name>
```

Comment:
I only revised chapter 3.7 not the whole thing, so what about just adding my name here?

Reply:
In any case, by the initial logic of this PR, @gucl-mu would need to be included here as well.

Reply:
@riedde @gucl-mu What do you think?

Reply:
@musicEnfanthen Before this discussion I was not sure what this responsibility means. When this chapter is in the responsibility of the Chairmen, then @gucl-mu and @riedde are the persons to be namend here.

Reply:
@riedde It is at least in the responsibility of the "main editors". That don't need to be the IG chairs necessarily but could be delegated as well. But it also makes sense to have this role connected to the IG chairs. 

Maybe you can have a short discussion with everyone involved (@margrethesb , @KristinaRichts and @gucl-mu) if they all would be OK with the change? (Don't expect any objections but think that it would be a matter of fairness.)

@kepper Any additions from your side?

Reply:
I talked to @gucl-mu and he is fine with the process.

Summarizing: Responsible for chapter 3 is @music-encoding/ig-metadata represented by the co-chairs or deligates.

@musicEnfanthen If no one, mentioned in this PR, has additions or contra arguments this is the way we do it for MEI 5.1